File: spark-yun-api/src/main/java/com/isxcode/star/api/pojos/work/res/RunWorkRes.java
package com.isxcode.star.api.pojos.work.res;

import java.util.List;
import lombok.AllArgsConstructor;
import lombok.Data;

@Data
@AllArgsConstructor
public class RunWorkRes {

  private String message;

  private String log;

  private List<List<String>> data;

  private String applicationId;

  private String yarnApplicationState;

  private String finalApplicationStatus;

  private String trackingUrl;
}

File: spark-yun-website/src/types/woks/res/RunWorkRes.ts
import { type BasePaginationRes } from '../../base/BasePaginationRes'
import { WorkRow } from '../info/WorkRow'

export interface RunWorkRes {
  message?: string
  log?: string
  yarnLog?: string
  data: [[]]
  applicationId?: string
  yarnApplicationState?: string
  finalApplicationStatus?: string
  trackingUrl?: string
}
